Source/WebCore/platform/video-codecs/BitReader.cpp

```diff
@@ -60,4 +60,19 @@ std::optional<bool> BitReader::readBit()
     return value;
 }
 
+void BitReader::skip(size_t bits)
+{
+    auto remainingBits = m_remainingBits;
+    bits -= remainingBits;
```

Comment:
This seems to imply that bits is greater than remainingBits.
Can we fix this? And add unit tests?
Maybe add skipBits and skipBytes also?

Reply:
`m_remainingBits` is the number of bits left in the current byte, not the number of total bits remaining.
So yet, bits can (and almost always will be here) greater than remainingBits